pkgs/os-specific/linux/guvcview/default.nix

```
@@ -1,5 +1,5 @@
 { config, stdenv, fetchurl, intltool, pkgconfig
-, gtk3, portaudio, SDL2, ffmpeg, udev, libusb1, libv4l, alsaLib, gsl
+, gtk3-x11, portaudio, SDL2, ffmpeg, udev, libusb1, libv4l, alsaLib, gsl, wrapGAppsHook
```

why do we have to use gtk3-x11?

---

According to `nix-locate`, this is the only package that has the file `org.gtk.Settings.FileChooser.gschema.xml`, as opposed to `gtk3`.

---

If you build `gtk3` you will quickly find this schema (and gtk3 wouldn't function properly without it)
```suggestion
, gtk3, portaudio, SDL2, ffmpeg, udev, libusb1, libv4l, alsaLib, gsl, wrapGAppsHook
```

---

You were right. `gtk3` and `gtk3-x11` seems to be the same for me as `nix-build -A` had to do nothing with this change.